### DISCUSSION TITLE
[uploaded.to] Some file load very slow on pyload

### DISCUSSION THREAD

**Discussion by kvoit** [pyload/pyload#3980] [Help]
Does anyone else have the problem that some files load very slowly with an uploaded.to premium account, and sometimes even stall. I've had this issue for at least a few weeks now. 
Other files load with > 20MB/s, and restarts usually solve this problem. I am not sure if this is a pyload or uploaded.to issue, but if multiple people have this issue, pyload could at least react on this by autorestarting slow downloads.

It is pretty difficult to give more info on this, since even the debug log looks normal.

**Comment by GammaC0de** [comment]
There was once a `RestartSlow` plugin, I don't know what happened to it though..

**Comment by GammaC0de** [comment]
Here it is: https://github.com/pyload/pyload/tree/23b49f52109942bd8cc121818d054ea134c63e64/module/plugins/hooks/RestartSlow.py

**Comment by Velociraptor85** [comment]
@kvoit did you try the RestartSlow Plugin?
